src/components/Input/Input.tsx

```diff
@@ -22,6 +23,7 @@ import Ref from '../Ref'
 
 export interface IInputProps {
   as?: any
+  children?: ReactChildren
```

Comment:
actually, this is not correct - please, consider this comment that explains why

Reply:
This is our custom ReactChildren so it is actually the correct one.